lib/ui/painting.dart

```diff
@@ -1091,9 +1091,8 @@ class Paint {
   /// Constructs an empty [Paint] object with all fields initialized to
   /// their defaults.
   Paint() {
-    if (enableDithering) {
-      _dither = true;
-    }
+    // TODO(matanlurey): Remove as part of https://github.com/flutter/flutter/issues/112498.
+    _enableDithering();
```

Comment:
Do we need to leave this in for now? Could we just rip it out entirely?

Reply:
We could rip it out with a much larger change, but I'm trying to do it piecemeal (i.e. https://github.com/flutter/engine/pull/46746).